cypress/integration/unit/decision.spec.js

```diff
@@ -88,11 +88,49 @@ context('Add files to an agenda', () => {
       folder: 'files', fileName: 'test', fileExtension: 'pdf',
     });
 
+    cy.get('@docCards').eq(0)
+      .within(() => {
+        cy.get('.vl-title--h6 > span').contains(/BIS/);
+      });
+
+    // Delete the TER piece, the BIS should then become the report
+    cy.addNewPieceToSignedDocumentContainer('test', {
+      folder: 'files', fileName: 'test', fileExtension: 'pdf',
+    });
+
+    cy.get('@docCards').should('have.length', 1);
+    cy.get('@docCards').eq(0)
+      .within(() => {
+        cy.get('.vl-title--h6 > span').contains(/TER/);
+        cy.get('.js-vl-accordion > button').click();
+        cy.get('.vl-accordion__panel > .vlc-document-card-item').as('pieces');
+        cy.get('@pieces').eq(0)
+          .within(() => {
+            cy.get('.ki-delete').click();
+          });
+      });
+
+    cy.get('.vl-modal').within(() => {
+      cy.get('button').contains('Verwijderen')
+        .click();
+    });
```

Comment:
Kunnen we hier niet best selectors gebruiken? Vooral voor de icons lijkt het me niet echt een goed idee om die er hardcoded in te zetten. Beste de icon's misschien ook in een aparte selector file te steken?

Reply:
Dit komt op verschillende plaatsen voor. Lijkt me momenteel ook geen showstopper. 
Mss klein improvement ticketje loggen voor cy.get('.vl-modal') selector

Reply:
https://kanselarij.atlassian.net/browse/KAS-1915